src/components/TodoList.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  CSSTransition,
+  TransitionGroup,
+} from 'react-transition-group';
+
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+import '../styles/animation.scss';
+
+type Props = {
+  todos: Todo[];
+  onRemove: (userId: number) => void;
```

Comment:
The `onRemove` function should have a parameter name `todoId` instead of `userId` to better represent its purpose.